### DISCUSSION TITLE
Repeater Component

### DISCUSSION THREAD

**Discussion by markdwoods3** [Budibase/budibase#1215] [Help] [answered]
Hey all, 

I'm running through the tutorial (scene 5) and on the Stacked List I've got stuck as it doesnt let me complete right across the whole bar. The 'text' fields are missing for me so I cant show a complete card. 

Probably something I'm doing wrong  but I'd be grateful for guidance on the point. 

Screenshots included for ease of reference. 

MW
![my app](https://user-images.githubusercontent.com/79729779/109369265-b2c96200-7893-11eb-9198-e5fc8d50cb83.png)
![tutorial](https://user-images.githubusercontent.com/79729779/109369267-b5c45280-7893-11eb-862f-88d903910aab.png)

**Comment by aptkingston** [comment, marked as answer]
Hi @markdwoods3. We updated the `StackedList` component and removed a few of those text settings, but the tutorial hasn't been updated to reflect this so thanks for letting us know. If you'd like a similar looking component to the old `StackedList`, you could use a `Container` component with `flex-direction: space-between` and add a few `Paragraph` components inside it. You can create pretty much any layout you like by using the design panel, and those prebuilt components are just designed to save time on some common designs.
